general/open-source.md

```diff
@@ -6,11 +6,25 @@ Open source software and open standards are critical components of modern web ap
 We believe relying upon open standards and software has been fundamental to the success of the software industry as a whole. To aid in this, Langley Foxall is committed to giving back to the community. We try to contribute useful open source software, libraries, and other tools wherever possible. These packages are often used internally or as a part of our bespoke software solutions.
 
 In fact, the page you are reading right now is part of just one of these open source projects.
+
 ## Our Open Source Projects
 
 We love contributions and want our packages to help people, so if you have suggestions for improvements, please file
 an issue on the relevant repository.
 
+## Creating Open Source Projects
+
+When creating open source software, we're aware that these projects can be used in systems that other developers build & rely on, because of this, we have rigid requirements for our oss which prevent breaking changes from being introduced into stable versions of the public projects we release.
```

Comment:
```suggestion
When creating open source software, we're aware that these projects can be used in systems that other developers build & rely on, because of this, we have rigid requirements for our OSS which help mitigate breaking changes from being introduced into stable versions of the public projects we release.
```

The suggestion above corrects a case typo, and also changes the working from 'prevent breaking changes' to 'help mitigate breaking changes', as this is not a guarantee anyone can 100% provide.